source/_patterns/02-organisms/components/highlight.mustache

```diff
@@ -0,0 +1,14 @@
+<div class="highlight" role="region" data-behaviour="Highlight"{{#headingId}} aria-labelledby="{{headingId}}"{{/headingId}}>
```

Comment:
If `headingId` is needed then we should add it to the schema. @discodavey?

Reply:
I removed the headingId, it is not needed